### ISSUE TITLE
Select `onChange` causes warning: `label` of `value` is not same as `label` in Select options.

### ISSUE THREAD

**Issue by phatify** [ant-design/ant-design#40205]
### What problem does this feature solve?

we can add the prefix icon in select option in some case needed

### What does the proposed API look like?

```
<Select options={[
   {
       prefixIcon: <AppleIcon />,
       label: 'Apple',
       value: 'apple'
   }
]} />
```

or

```
<Select>
   <Option prefixIcon={<AppleIcon />} value="apple">
        Apple
   </Option>
</Select>
```

**Comment by MadCcc** [comment]
Why not wrap prefix in label？

**Comment by MadCcc** [comment]
Could you provide a reproduction?

**Comment by MadCcc** [comment]
Using `optionLabelProp="label"` could avoid this warning.

**Comment by MadCcc** [comment]
It should be a bug.

**Comment by liorEhrlich** [comment]
+1

**Comment by qianzhong516** [comment]
> > Could you provide a reproduction?
> 
> @MadCcc Of course! You can check it: https://stackblitz.com/edit/nextjs-esfw33?file=pages/index.js

This is not a bug. The error literally tells you what's gone wrong
> label of value is not same as label in Select options.

There is a mismatch between the values and labels in your code because you didn't specify `value={fruit}` in `Select`.

**Comment by denvudd** [comment]
+1

**Comment by egor2000sleptsov** [comment]
+1

**Comment by therbta** [comment]
+1

**Comment by DeveloperFromUkraine** [comment]
+1

**Comment by Dinika** [comment]
Is this issue fixed? I am on antd `5.4.7` and rc-select `~14.10.0` but I still see the warning in unit tests.
